interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -159,168 +159,65 @@ <h2>{xlt t="Vitals"}&nbsp;&nbsp;&nbsp;<a href="../summary/demographics.php" clas
-                                    {foreach item=result from=$results}
-                                            <td class='historicalvalues'>{if $result.head_circ != 0}{math equation="number * constant" number=$result.head_circ constant=2.54 format="%.2f"|text}{/if}</td>
-                                    {/foreach}</tr>
-
-                                {if $units_of_measurement == 4 || $gbl_vitals_options > 0}<tr class="hide">{else}<tr>{/if}
-                                        {if $units_of_measurement == 2}<td class="unfocus graph" id="waist_circ">{else}<td class="graph" id="waist_circ">{/if}{xlt t="Waist Circumference"}</td>
-                                    {if $units_of_measurement == 2}<td class="unfocus">{else}<td>{/if}{xlt t="in"}</td>
-                                    {if $units_of_measurement == 2}<td class="valuesunfocus">{else}<td class='currentvalues p-2'>{/if}
-                                            <input type="text" class="form-control" size='5' name='waist_circ' id='waist_circ_input' value="{if $vitals->get_waist_circ() != 0}{$vitals->get_waist_circ()|attr}{/if}" onChange="convIntoCm('waist_circ_input');"/>
-                                            </td>
-                                {foreach item=result from=$results}
-                                    <td class='historicalvalues'>{if $result.waist_circ != 0}{$result.waist_circ|text}{/if}</td>
-                                {/foreach}</tr>
-
-                                {if $units_of_measurement == 3 || $gbl_vitals_options > 0}<tr class="hide">{else}<tr>{/if}
-                                    {if $units_of_measurement == 1}<td class="unfocus graph" id="waist_circ_metric">{else}<td class="graph" id="waist_circ_metric">{/if}{xlt t="Waist Circumference"}</td>
-                                    {if $units_of_measurement == 1}<td class="unfocus">{else}<td>{/if}{xlt t="cm"}</td>
-                                        {if $units_of_measurement == 1}<td class="valuesunfocus">{else}<td class='currentvalues p-2'>{/if}
-                                        <input type="text" class="form-control" size='5' id='waist_circ_input_metric' value="{if $vitals->get_waist_circ() != 0}{math equation='round(number * constant,1)' number=$vitals->get_waist_circ() constant=2.54 format='%.2f'|attr}{/if}" onChange="convCmtoIn('waist_circ_input');"/>
-                                        </td>
-                                        {foreach item=result from=$results}
-                                        <td class='historicalvalues'>{if $result.waist_circ != 0}{math equation="number * constant" number=$result.waist_circ constant=2.54 format="%.2f"|text}{/if}</td>
-                                        {/foreach}</tr>
+                                {include file="vitals_textbox_conversion.tpl" title="Weight" input="weight"
+                                    vitalsValue=$vitals->get_weight() vitalsValueMetric=$vitals->get_weight_metric()
```

Comment:
do the vitalsValue* things need quotes around them (would do in html, but this isn't straight html, so i don't know)

Reply:
Also need to ensure translations are happening

Reply:
Anything other than the file parameter defines a new variable in the included file.  So I'm creating new variables for title, input, vitalsValue, and vitalsValueMetric so they don't need quotes around them.